tests/kernel/ib_tests.pm

```diff
@@ -119,10 +116,13 @@ sub run {
 
     use_ssh_serial_console;
 
+    # unload firewall. MPI- and libfabric-tests require too many open ports
+    script_run("systemctl stop firewalld");
+
     if ($role eq 'IBTEST_MASTER') {
         ibtest_master;
     }
-    elsif ($role eq 'IBTEST_SLAVE1' || $role eq 'IBTEST_SLAVE2') {
+    elsif ($role eq 'IBTEST_SLAVE') {
         ibtest_slave;
     }
 
```

Comment:
not sure power_action is needed, as OpenQA powers off the machines always after the tests